.github/workflows/pull-request.yml

```diff
@@ -119,6 +123,10 @@ jobs:
       - name: build testbench
         run: ./scripts/rebuild-testbench.sh
 
+      - name: load octave packages
+        run: echo "more off" >> .octaverc; echo "pkg load signal" >> .octaverc;
+             echo "pkg load io" >> .octaverc
```

Comment:
If I need to reproduce a problem found by CI then I will need these commands too? Then how about this instead:

```
install ~/tools/testbench/load.octave >> ~/.octaverc
```

Reply:
I created .octave file in tools/test/audio, how about that?